### PR TITLE
Add environment secrets to release workflow

### PR DESCRIPTION
**Pull request by justinkaseman**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -98,10 +101,8 @@ jobs:
       - name: Configure AWS Credentials for SDLC Public ECR
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_PUBLICECR_ACCESSKEY }}
-          aws-secret-access-key: ${{ secrets.AWS_PUBLICECR_SECRETKEY }}
           aws-region: ${{ secrets.AWS_REGION }}
-          role-to-assume: ${{ secrets.AWS_PUBLICECR_ROLE_ARN }}
+          role-to-assume: ${{ secrets.AWS_ARN_EA_PUBLIC }}
```
Comment (HenryNguyen5) [inline .github/workflows/release.yml:105]: Any particular reason for rename? The previous one was much more descriptive

**Review by HenryNguyen5 — CHANGES_REQUESTED** [review]

**Review by chainchad — DISMISSED** [review]
LGTM but I agree with [HenryNugyen5's comment](https://github.com/smartcontractkit/external-adapters-js/pull/1947/files#r895930943) about the name (nit).

**Review by HenryNguyen5 — DISMISSED** [review]

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -57,8 +61,7 @@ jobs:
       - name: Configure AWS Credentials for SDLC Private ECR
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_PRIVATEECR_ACCESSKEY }}
-          aws-secret-access-key: ${{ secrets.AWS_PRIVATEECR_SECRETKEY }}
+          aws-role-to-assume: ${{ secrets.AWS_PRIVATE_ECR_ARN }}
```
[the remark this answers is not in the document]
Comment (HenryNguyen5) [reply .github/workflows/release.yml:64]: I think `AWS_OIDC_IAM_ROLE_ARN` is ideal, @chainchad should we document this std practice we want to adopt? Namely, "AWS_OIDC_IAM_ROLE_ARN is used as a secret per environment, where it contains the role arn containing the relevant permissions for the github environment it lives in"

[the remark this answers is not in the document]
Comment (chainchad) [reply .github/workflows/release.yml:64]: Sounds good, just put up a separate PR in another repo and added you as a reviewer for the docs on that @HenryNguyen5.

Comment (chainchad) [inline .github/workflows/release.yml:64]:
```suggestion
          aws-role-to-assume: ${{ secrets.AWS_OIDC_IAM_ROLE_ARN }}
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -98,10 +101,8 @@ jobs:
       - name: Configure AWS Credentials for SDLC Public ECR
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_PUBLICECR_ACCESSKEY }}
-          aws-secret-access-key: ${{ secrets.AWS_PUBLICECR_SECRETKEY }}
           aws-region: ${{ secrets.AWS_REGION }}
-          role-to-assume: ${{ secrets.AWS_PUBLICECR_ROLE_ARN }}
+          role-to-assume: ${{ secrets.AWS_PUBLIC_ECR_ARN }}
```
Comment (chainchad) [inline .github/workflows/release.yml:105]:
```suggestion
          role-to-assume: ${{ secrets.AWS_OIDC_IAM_ROLE_ARN }}
```

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -70,9 +73,9 @@ jobs:
           branch: null
           latest: null
 
-          image-prefix: ${{ secrets.SDLC_ACCOUNT_ID }}.dkr.ecr.${{ secrets.AWS_REGION }}.amazonaws.com/adapters/
+          image-prefix: ${{ secrets.SDLC_ACCOUNT_ID }}.dkr.ecr.${{ secrets.AWS_REGION_ECR_PRIVATE }}.amazonaws.com/adapters/
           adapter-name: ${{ matrix.adapter.name }}
-          aws-region: ${{ secrets.AWS_REGION }}
+          aws-region: ${{ secrets.AWS_REGION_ECR_PRIVATE }}
           aws-ecr-private: 'true' # Use of string intentional for evaluation
           aws-ecr-account-ids: ${{ secrets.AWS_PRIVATE_ECR_SECONDARY_ACCOUNT_ACCESS_IDS }}
```
Comment (justinkaseman) [inline .github/workflows/release.yml:80]: @HenryNguyen5 we will need your help with this one

**Review by chainchad — COMMENTED** [review]
LGTM but I'd wait to merge until the separate role PR gets merged.

**Review by chainchad — APPROVED** [review]
👍

**Pull request merged**